index.js

```diff
@@ -37,3 +37,22 @@ exports.co = function(generator) {
     });
   });
 };
+
+// promote :: (a -> (Error -> b -> ()) -> ()) -> (a -> Promise b)
+exports.promote = function(nodeFunction) {
+  return arity(nodeFunction.length - 1, () => {
+    return new Promise((resolve, reject) => {
+      const $functionArgs = Array.prototype.slice.call(arguments);
+      $functionArgs.push(() => {
+        const callbackArgs = Array.prototype.slice.call(arguments);
+        const error = callbackArgs[0];
+        if (error != null) {
+          reject(error);
+        } else {
+          resolve(callbackArgs.slice(1));
```

Comment:
Have you considered adding Ramda as a dependency? `Array.prototype.slice.call(arguments).slice(1)` would become `R.tail(arguments)`.